### PR TITLE
Travis coveralls

### PR DESCRIPTION
**Pull request by adrosenbaum**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -51,39 +51,39 @@ def parse_reqs(req_path='./requirements.txt'):
...
-    min_hk_version = int(HOUSEKEEPER_VERSION.replace('.',''))
-
-    if hk_version >= min_hk_version:
-
-        return True
-
-
-if not check_scout(): sys.exit('Dependency problem: scout >= {} is required'.format(SCOUT_VERSION))
-if not check_housekeeper(): sys.exit('Dependency problem: housekeeper >= {} is required'.format(HOUSEKEEPER_VERSION))
+# def check_scout():
```
Comment (henrikstranneheim) [inline setup.py]: Better to make a switch than to comment this out

File: mutacc_auto/commands/mutacc_command.py [hunk 2]
```diff
@@ -20,7 +20,7 @@ def __init__(self, config_file, padding, case_file):
         super(MutAccExract, self).__init__(config_file)
 
         self.add_subcommand('extract')
-        self.add_option('padding', value=padding)
+        self.add_option('padding', value=str(padding))
```
Comment (henrikstranneheim) [inline mutacc_auto/commands/mutacc_command.py:23]: Why is padding a string and not an int

File: tests/cli/test_import_command.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
+import pytest
+from mock import patch
+import yaml
+from pathlib import Path
+
+from click.testing import CliRunner
+
+from mutacc_auto.cli.root import cli
+
+INPUTS_LIST = [{'input_file': 'path_to_file', 'padding':600}]*2
```
Comment (henrikstranneheim) [inline tests/cli/test_import_command.py:10]: What does "*2" do? A comment would be nice

Comment (henrikstranneheim) [reply tests/cli/test_import_command.py:10]: 600 seems like a constant. Could you use named constants to make the code easier to understand. For instance why 600 and not 300?

**Review by henrikstranneheim — COMMENTED** [review]
Nice!

File: mutacc_auto/commands/mutacc_command.py (hunk 2 above)
Comment (adrosenbaum) [reply mutacc_auto/commands/mutacc_command.py:23]: Converts to string, the complete command line command will ultimately be given as a string, i.e. 'mutacc --config-file ... extract --padding 123 --case ...'

File: tests/cli/test_import_command.py (hunk 3 above)
Comment (adrosenbaum) [reply tests/cli/test_import_command.py:10]: "*2" just duplicates the elements in the list, so test is run with a list of two elements (although identical). Can make 'padding' a constant, Although here the value has no actual importance, this is just to test the function.

**Pull request merged**